CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ NOTE: This changes the Go runtime "GOMAXPROCS" to 1. This is done to limit the
   race condition problem in the Linux kernel (#2500) and parallel IO issues
   on nodes with high numbers of CPUs/CPU threads (#1880).
 
+NOTE: A command line arg has been changed from `--web.config` to `--web.config.file`.
```

Comment:
It seems like the commit history is a bit messed up. Would you mind fixing/squashing your commits so we don't have unrelated changes here?

Reply:
Should be fixed now